src/Notifier.js

```diff
@@ -98,8 +105,10 @@ const Notifier = {
         this.boundOnRoomTimeline = this.onRoomTimeline.bind(this);
         this.boundOnSyncStateChange = this.onSyncStateChange.bind(this);
         this.boundOnRoomReceipt = this.onRoomReceipt.bind(this);
+        this.boundOnEventDecrypted = this.onEventDecrypted.bind(this);
         MatrixClientPeg.get().on('Room.timeline', this.boundOnRoomTimeline);
         MatrixClientPeg.get().on('Room.receipt', this.boundOnRoomReceipt);
+        MatrixClientPeg.get().on('Event.decrypted', this.boundOnEventDecrypted);
```

Comment:
We can, although we've mostly been sticking with the pattern of listening for everything via the client (for better or worse), and we'd have to worry about leaking event listeners if they were never decrypted.